components/common/footer.js

```diff
@@ -12,12 +12,12 @@ const loggedOutLinks = [
     text: 'About Us'
   },
   {
-    url: '/how-it-works',
-    text: 'How It Works'
+    url: '/faq',
+    text: 'FAQs'
   },
   {
-    url: '/contact',
-    text: 'Contact Us'
+    url: '/how-it-works',
+    text: 'How It Works'
```

Comment:
idk, i like it when websites have all their links in the footer. not sure of any reason to hide/show based on context

Reply:
Changed it so the footer links are the same logged in and logged out (with the exception of "Log In" and "Sign Up")